origin-growth/src/rules/rules.js

```diff
@@ -1,11 +1,59 @@
 const Sequelize = require('sequelize')
+const BigNumber = require('bignumber.js')
 
 const db = require('../models')
-const { GrowthEventTypes, GrowthEventStatuses } = require('../enums')
+const {
+  GrowthEventTypes,
+  GrowthEventStatuses,
+  GrowthCampaignStatuses,
+  GrowthActionStatus,
+  GrowthActionType } = require('../enums')
 
 // System cap for number of rewards per rule.
 const MAX_NUM_REWARDS_PER_RULE = 1000
 
+const sumUpRewards = (rewards) => {
+  if (rewards === null || rewards.length === 0){
+    return null
+  }
+
+  const finalReward = rewards.reduce((first, second) => {
+    if (first.currency !== second.currency)
+      throw new Error(`At least two rewards have different currencies. ${first.currency} ${second.currency}`)
+    return {
+      amount: BigNumber(first.amount).plus(BigNumber(second.amount)),
+      currency: first.currency
+    }
+  })
+
+  return {
+    amount: finalReward.amount.toString(),
```

Comment:
Note that all rules amount are in natural unit so the sum will be in natural unit as well.
e.g. for 1 OGN, as opposed to showing 1000000000000000 it would return 1.

Should those be converted to token unit here or will you be handling this at a different layer ?

Reply:
Currently the frontend receives natural units and queries the wallet for decimal places of the token's currency. Then converts to token units. I would keep it this way?

Reply:
Yes, I think what you have is better. Let's keep it this way.